runtime/syntax/testdir/runtest.vim

```diff
@@ -145,6 +147,23 @@ func RunTest()
 	  redraw!
 	endfunc
 
+	func SetUpVim()
+	  call cursor(1, 1)
+	  " Defend against rogue TEST_SETUP commands.
+	  for _ in range(20)
+	    let lnum = search('\<TEST_SETUP\>', 'eW', 20)
+	    if lnum < 1
+	      break
+	    endif
+	    exe substitute(getline(lnum), '.*TEST_SETUP', '', '')
+	  endfor
```

Comment:
Is it possible to execute "dangerous" commands this way? Should this be in the sandbox, if it isn't already?

Reply:
It is sure possible to _try_ to execute a non-command and  
fall flat.  Does the sourcing of `syntax/foo.vim` happen  
in a sandbox?  If not, you may well sandbox the setting  
of an innocuous variable _here_ and defer arbitrary execution  
to the code that queries it _there_.  The author(s) of  
`indent/testdir/runtest.vim` did not make any allowance for  
"better securing" similar lines.

Let us seek more informed opinions on the severity of doing  
nothing.

Reply:
that should be fine, since this is only run from the test-suite. So hopefully any dangerous commands are noticed before we introduce them into the repo.

Reply:
A unique name is a boon.

I'll go and rebase commits then, replace `TEST_SETUP` with  
`VIM_TEST_SETUP`, reflow documentation, regenerate and peruse  
`dumps/c_*`.

@chrisbra, the CI jobs seem to no longer fail for `runtime/`  
tests.  I guess, the fix for _italics_ have worked, right?

Reply:
yes looks like it. But I think we should move it into the `Xtestscript` instead?